### ISSUE TITLE
Bug Report: Missing Requirements File for xhal?

### ISSUE THREAD

**Issue by bdorney** [cms-gem-daq-project/xhal#57]
## Brief summary of issue

On the P5 machine I've come across a few problems trying to use the v3 electronics.  The release instructions are not successful see updated comments in issue #55. 

I have been using the `cc7` virtual machine `gemvm-daqcc7` 

After compiling all libraries on `gem904qc8daq` (os: `cc7`) and placing them in:

```
$XHAL_ROOT/lib/arm
$XHAL_ROOT/lib/x86_64
```

Trying to run `reg_interface.py` seems to generate the following import errors:

```
ImportError: No module named lxml.etree
ImportError: No module named cmd
```

The `vm` mentioned and the `cc7` DAQ machines in 904 seem to use `python v2.7.5` while the daq machines in 904 seem to have these packages installed by default? 

### Types of issue

- [X] Bug report (report an issue with the code)
- [ ] Feature request (request for change which adds functionality)

## Expected Behavior

We should have a requirements file to assist with system installation.

## Current Behavior

No requirements file is present.

## Possible Solution (for bugs)

Add a requirements file.

## Context (for feature requests)

Takes awhile to setup the code or a new release.

## Your Environment

* Version used: b4980d841b4738740d5bc7c25bda52e537dec17d
* Shell used: /bin/bash

**Comment by mexanick** [comment]
requirements would be handled in a future packaging release

**Comment by bdorney** [comment]
Was this closed by #73 @mexanick ?

**Comment by mexanick** [comment]
yes
